src/sentry/event_manager.py

```diff
@@ -820,7 +820,7 @@ def save(self, project_id, raw=False, assume_normalized=False):
 
         transaction_name = data.get('transaction')
         logger_name = data.get('logger')
-        fingerprint = data.get('fingerprint')
+        fingerprint = data.get('fingerprint') or ['{{ default }}']
```

Comment:
Seems odd to add the default here, and also handle it in `calculate_event_hashes`. Might make sense to remove this right hand side of the `or` expression?

Reply:
@tkaemming i actually want to write it back into the data though. That's why i left it here and in the other branch as well.